### PR TITLE
Remove .min file from bower.json main section

### PR DESCRIPTION
**Pull request by david-poirier-csn**
main section should not include minified files - https://github.com/bower/bower.json-spec

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -4,8 +4,7 @@
     "homepage": "https://github.com/aspnet/jquery-validation-unobtrusive",
     "description": "Add-on to jQuery Validation to enable unobtrusive validation options in data-* attributes.",
     "main": [
-        "jquery.validate.unobtrusive.js",
-        "jquery.validate.unobtrusive.min.js"
+        "jquery.validate.unobtrusive.js"
     ],
     "ignore": "",
```
Comment (kichalla) [inline bower.json]: Could you also add the necessary files to the `ignore` list?

BTW would you be interested in doing similar changes for this package too? https://github.com/aspnet/jquery-ajax-unobtrusive/blob/master/bower.json#L6

**Comment by david-poirier-csn** [comment]
No worries, I added json, md and txt files to ignore. I also made the same type of changes to the jquery-ajax-unobtrusive bower.json

File: bower.json [hunk 2]
```diff
@@ -4,10 +4,13 @@
     "homepage": "https://github.com/aspnet/jquery-validation-unobtrusive",
     "description": "Add-on to jQuery Validation to enable unobtrusive validation options in data-* attributes.",
     "main": [
-        "jquery.validate.unobtrusive.js",
-        "jquery.validate.unobtrusive.min.js"
+        "jquery.validate.unobtrusive.js"
+    ],
+    "ignore": [
+        "*.json",
```
Comment (kichalla) [inline bower.json]: should add `.gitignore`..no? BTW I noticed that even though we try to ignore `bower.json` here it still gets installed by bower when installing a package.

Thanks! for updating the other repo too

Comment (david-poirier-csn) [inline bower.json]: I’ve added an ignore for dot (.) files and folders. Not sure about bower.json, probably a feature ☺

**Comment by kichalla** [comment]
:shipit:

**Comment by kichalla** [comment]
20b7833790a43e456cb53f6052c88ca258c57391

Thanks for the contribution! @david-poirier-csn

**Pull request closed without merging**